pkg/calculator/calculator_test.go

```diff
@@ -0,0 +1,29 @@
+package calculator
+
+import (
+	"fmt"
+	"testing"
+
+	v1 "github.com/re-cinq/cloud-carbon/pkg/types/v1"
+)
+
+func TestCalculateEmissions(t *testing.T) {
+	c := calculate{
+		cores:    4,
+		usage:    v1.Percentage(24),
+		minWatts: 1.3423402398570,
+		maxWatts: 4.00498247528,
+		chip:     35.23458732,
+		pue:      1.0123,
+		gridCO2e: 0.00023,
+	}
+
+	fmt.Println(c.operationalEmissions(5))
+}
+
+func TestCalculateEmbodiedEmissions(t *testing.T) {
+	c := calculate{
+		totalEmbodied: 1706.48,
+	}
+	fmt.Println(c.embodiedEmissions(5))
```

Comment:
Do we want to print here or test for value?

Reply:
we want to test for value of course. I've created an issue to add tests for the calculator, I just don't want this PR to get too big. I will also tie it all into the rest of the project in the next PR as well. So it's stand alone and not being used yet - hence why I think it's okay to merge without tests just yet.